core/input/input.cpp

```diff
@@ -1013,6 +1128,291 @@ void Input::vibrate_handheld(int p_duration_ms, float p_amplitude) {
 	OS::get_singleton()->vibrate_handheld(p_duration_ms, p_amplitude);
 }
 
+/*
+	PS5 trigger effect documentation:
+	https://controllers.fandom.com/wiki/Sony_DualSense#FFB_Trigger_Modes
+*/
```

Comment:
I think this copyrighted code should be moved elsewhere to make the copyright more manageable here